bitnami/pgbouncer/1/debian-11/rootfs/opt/bitnami/scripts/pgbouncer-env.sh

```diff
@@ -126,6 +127,7 @@ export PGBOUNCER_AUTH_HBA_FILE="${PGBOUNCER_AUTH_HBA_FILE:-}"
 export PGBOUNCER_STATS_USERS="${PGBOUNCER_STATS_USERS:-}"
 export PGBOUNCER_POOL_MODE="${PGBOUNCER_POOL_MODE:-}"
 export PGBOUNCER_INIT_SLEEP_TIME="${PGBOUNCER_INIT_SLEEP_TIME:-10}"
+export PGBOUNCER_SERVER_FAST_CLOSE="${PGBOUNCER_SERVER_FAST_CLOSE:-**0**}"
```

Comment:
Default should be just 0, without ** **, shouldn't it?

Reply:
> Default should be just 0, without ** **, shouldn't it?

Yeah, we ran into this as well. We overrode the env var in our K8s chart (to just "0") and it fixed the issue.